pythonvectordbceph.py

```diff
@@ -116,11 +126,27 @@ def pythonvectordbappceph():
         case "TEXT":
             object_content = object_data["Body"].read().decode("utf-8")
             objectlist = []
-            objectlist.append(object_content)
-            # default embedding function provided by milvus, it has some size limtation for the object
-            # embedding_fn = milvus_model.DefaultEmbeddingFunction() #dimension 768
+            if chunk_size > 1:
+                object_size = object_data["ContentLength"]
+                if object_size == 0 :
+                    app.logger.debug("object size zero cannot be chunked")
+                    return
+                text_splitter = CharacterTextSplitter(
+                    separator=".",
```

Comment:
do you split by `.` or by size?

Reply:
is it possible to demo chunking done by content (by the language model itself)?

Reply:
> do you split by `.` or by size?

First check for ".", if not then chunking happen based on size